packages/supplementary-contracts/contracts/plonk_verifier/ISP1Verifier.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+/// @title SP1 Verifier Interface
+/// @author Succinct Labs
+/// @notice This contract is the interface for the SP1 Verifier.
```

Comment:
Is it possible to `pnpm install` the SP1 repo as a dependencies to avoid copying the code?

Reply:
See https://github.com/taikoxyz/taiko-mono/pull/17878/files

Reply:
Technically we can, but this is the only file needed from that repo. All the other files are our custom - to facilitate easier devnet E2E testing AFAIK.

Reply:
Previously we have a "third_party" folder, now it's all gone as all external file are dependencies managed by npm. I hope we don't have to bring "third_party" back.